### PR TITLE
get values from dedicated events-scrape secret

### PR DESCRIPTION
**Pull request by michaellevy101**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,3 +26,5 @@ This tool can be used by `events_scrape` cli command (if installed) or running u
 | `ASSISTED_SERVICE_URL`| Assisted service url  | https://api.openshift.com |
 | `OFFLINE_TOKEN`       | Assisted service offline token  | |
 | `BACKUP_DESTINATION`  | Path to save backup, if empty no backups will be saved  | |
+| `SSO_URL`             | SSO server URL  | |
+
```
Comment (osherdp) [inline README.md:30]: seems unused, no?

**Review by osherdp — CHANGES_REQUESTED** [review]
Can you add some description to the PR?
I'm not sure I understand how it will be used when deployed on the environment

File: README.md (hunk 1 above)
Comment (michaellevy101) [reply README.md:30]: used by the AI client

**Review by osherdp — APPROVED** [review]

**Pull request merged**